services/tutorials/ethereum/send-a-transaction/send-a-transaction-viem.md

```diff
@@ -0,0 +1,224 @@
+---
+description: Send a transaction using Viem.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Use Viem
+
+In this tutorial, you'll send a transaction of 0.001 ETH from one account to another using the [Viem](https://viem.sh/)
+TypeScript library.
+
+Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/)
+- [Install TypeScript](https://www.typescriptlang.org/download/)
+- [Install `ts-node`](https://www.npmjs.com/package/ts-node)
+- An Ethereum account
+
+:::info
+Use [MetaMask](https://metamask.io/) or similar to create an Ethereum account for testing.
+:::
+
+## Steps
+
+### 1. Select your network and verify funds
+
+- **Sepolia** -  To use the Sepolia testnet, ensure that your account has Sepolia ETH.
+    You can use the [MetaMask faucet](/developer-tools/faucet) to add more funds.
+- **Alternative network** - To use an alternative network, ensure that your account has testnet ETH
+    for that network.
+
+    :::note
+    When using an alternative network, update the chain name in the script (in Step 5) with the
+    alternative network name from the [Viem supported chains list](https://github.com/wevm/viem/blob/main/src/chains/index.ts).
+    :::
+
+### 2. Create a project directory
+
+Create a new directory for your project using the command line:
+
+```bash
+mkdir infura
+```
+
+Change into the new directory:
+
+```bash
+cd infura
+```
+
+### 3. Initialize the project
+
+Create a `package.json` file with default values:
+
+```bash
+npm init -y
+```
+
+Generate a `tsconfig.json` file, which is used to configure TypeScript compiler options:
+
+```bash
+tsc --init --resolveJsonModule true
+```
+
+### 4. Install the required packages
+
+Install the required packages in the project directory.
+
+```bash
+npm i viem
+```
+
+```bash
+npm install -D tslib @types/node
+```
+
+### 5. Create a `config.ts` file
+
+Create a `config.ts` file in your project directory to store the private key of the sending account:
+
+<Tabs>
+  <TabItem value="Syntax" label="Syntax" default>
+
+```tsx title="config.ts"
+import { privateKeyToAccount } from 'viem/accounts'
+export const account = privateKeyToAccount('<PRIVATE-KEY>')
+```
+  </TabItem>
+  <TabItem value="Example" label="Example" default>
+
+```tsx title="config.ts"
+import { privateKeyToAccount } from 'viem/accounts'
+export const account = privateKeyToAccount('0x561...x...61df')
```

Comment:
````suggestion
import { privateKeyToAccount } from "viem/accounts"
export const account = privateKeyToAccount("<PRIVATE-KEY>")
```
  </TabItem>
  <TabItem value="Example" label="Example" default>

```tsx title="config.ts"
import { privateKeyToAccount } from "viem/accounts"
export const account = privateKeyToAccount("0x561...x...61df")
````

Use double quotes in code samples.